pkg/controller/sparkapplication/sparkapp_util.go

```diff
@@ -145,6 +145,10 @@ func getExecutorContainerTerminatedState(podStatus apiv1.PodStatus) *apiv1.Conta
 	return state
 }
 
+func getDriverContainerRunningState(podStatus apiv1.PodStatus) *apiv1.ContainerStateRunning {
```

Comment:
Is this method really needed given it's only wrapping a one-line call to another method?

Reply:
Thanks, this method is lost.